src/sd-boot.h

```diff
@@ -31,6 +31,7 @@ typedef struct sdb_entry_data {
 	char	machine_id[SDB_LINE_MAX];
 	char	version[SDB_LINE_MAX];
 	char	options[SDB_LINE_MAX];
+	char	linux_[SDB_LINE_MAX];
```

Comment:
I assume the trailing underscore is there because linux is a cpp macro, but it looks odd :-)
Please consider using some other identifier like "kernel" or "image"